allennlp/predictors/biaffine_dependency_parser.py

```diff
@@ -1,12 +1,79 @@
+from typing import Dict, Any, List, Tuple
 
 from overrides import overrides
 
-from allennlp.common.util import JsonDict
+from allennlp.common.util import JsonDict, sanitize
 from allennlp.data import DatasetReader, Instance
 from allennlp.models import Model
 from allennlp.predictors.predictor import Predictor
 from allennlp.data.tokenizers.word_splitter import SpacyWordSplitter
 
+# POS tags have a unified colour.
+NODE_TYPE_TO_STYLE = {}
+
+NODE_TYPE_TO_STYLE["root"] = ["color5", "strong"]
+NODE_TYPE_TO_STYLE["dep"] = ["color5", "strong"]
+
+# Arguments
+NODE_TYPE_TO_STYLE["nsubj"] = ["color1"]
+NODE_TYPE_TO_STYLE["nsubjpass"] = ["color1"]
+NODE_TYPE_TO_STYLE["csubj"] = ["color1"]
+NODE_TYPE_TO_STYLE["csubjpass"] = ["color1"]
```

Comment:
nit: you could make this a lot shorter if you did something like

```
NODE_TYPE_TO_STYLE = {
    **{tag: ["color5", "strong"] for tag in ['root', 'dep']},
    **{tag: ["color1"] for tag in [...]},
    ...
}
```

but it's up to you